DXMainClient/DXGUI/Multiplayer/PlayerListBox.cs

```diff
@@ -0,0 +1,178 @@
+                    }
+                    else
+                    {
+                        drawnWidth = windowRectangle.Width - 2 - scrollBar.ClientRectangle.Width;
+                    }
+
+                    Renderer.FillRectangle(
+                        new Rectangle(windowRectangle.X + 1, windowRectangle.Y + height,
+                        drawnWidth, lbItem.TextLines.Count * LineHeight),
+                        GetColorWithAlpha(FocusColor));
+                }
+
+                if (user.IsAdmin)
+                {
+                    Renderer.DrawTexture(adminGameIcon,
+                        new Rectangle(windowRectangle.X + x, windowRectangle.Y + height,
+                        adminGameIcon.Width, adminGameIcon.Height), Color.White);
+                }
+                else
+                {
+                    if (user.IRCUser.GameID < 0 || user.IRCUser.GameID >= gameCollection.GameList.Count)
+                    {
+                        Renderer.DrawTexture(unknownGameIcon,
+                            new Rectangle(windowRectangle.X + x, windowRectangle.Y + height,
+                            unknownGameIcon.Width, unknownGameIcon.Height), Color.White);
+                    }
+                    else
+                    {
+                        var gameIconTexture = gameCollection.GameList[user.IRCUser.GameID].Texture;
+                        Renderer.DrawTexture(gameIconTexture,
+                        new Rectangle(windowRectangle.X + x, windowRectangle.Y + height,
+                        gameIconTexture.Width, gameIconTexture.Height), Color.White);
+                    }
+                }
+
+                x += adminGameIcon.Width + margin;
+
+                // Friend Icon
+                if (user.IsFriend)
+                {
+                    Renderer.DrawTexture(friendIcon,
+                        new Rectangle(windowRectangle.X + x, windowRectangle.Y + height,
+                        friendIcon.Width, friendIcon.Height), Color.White);
+
+                    x += friendIcon.Width + margin;
+                }
+
+                // Badge Icon - coming soon
+                /*
+                Renderer.DrawTexture(badgeGameIcon,
+                    new Rectangle(windowRectangle.X + x, windowRectangle.Y + height,
+                    badgeGameIcon.Width, badgeGameIcon.Height), Color.White);
+
+                x += badgeGameIcon.Width + margin;
+                */
+
+                // Player Name
+                string name = user.IsAdmin ? user.IRCUser.Name + " (Admin)" : user.IRCUser.Name;
+                x += lbItem.TextXPadding;
+
+                Renderer.DrawStringWithShadow(name, FontIndex,
+                    new Vector2(windowRectangle.X + x, windowRectangle.Y + height),
+                    lbItem.TextColor);
+
+                height += LineHeight;
+            }
+
+            if (DrawBorders)
+                DrawPanelBorders();
```

Comment:
You need to call DrawChildren at the end of the draw function for the scroll bar to get drawn.